schema/schema.sql

```diff
@@ -19,7 +19,7 @@ CREATE TABLE subscriber (
 CREATE TABLE artist (
     artist_id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
     name TEXT NOT NULL,
-    url TEXT UNIQUE NOT NULL
+    url TEXT NOT NULL
```

Comment:
Why should URL not be unique?

Reply:
To accommodate for URLS that different artists may share, e.g. record label pages, solo releases by band members.

Reply:
Oh okay I understand now, thanks